packages/components/src/internal/components/lineage/vis/VisGraph.tsx

```diff
@@ -169,7 +170,24 @@ export class VisGraph extends Component<VisGraphProps, VisGraphState> {
     };
 
     fitGraph = (): void => {
-        this.network.fit();
+        // NK: This is a arbitrarily set heuristic for deciding when to focus vs when to fit to
+        // the entire graph. I'm trying to delineate "small" graphs from "large" graphs.
+        if (this.network.body.nodeIndices.length > 50) {
```

Comment:
I agree this is arbitrary but seems like a good proxy for small/large. Can always adjust based on user feedback.